### DISCUSSION TITLE
err_ssl_protocol_error only for some users

### DISCUSSION THREAD

**Discussion by rpatel15-hue** [vercel/community#3827] [Help]
### Summary

A user is facing err_ssl_protocol_error on both mobile and desktop chrome, not sure what is going on, seems to be working for all other users. I asked to check with mobile data hoever this is important client and I am not sure how to debug this
using gdomains with vercel

website: www.pipnote.com

letsdebug is clean - https://letsdebug.net/www.pipnote.com/1596078

dnsviz is showing insecure for 
name.vercel-dns.com/A
cname.vercel-dns.com/A
cname.vercel-dns.com/A
cname.vercel-dns.com/A
https://dnsviz.net/d/www.pipnote.com/dnssec/

ssllabs: https://www.ssllabs.com/ssltest/analyze.html?d=www.pipnote.com
A+ for both

Not sure what else I can do.

### Steps to Reproduce

not sure

**Comment by amyegan** [comment]
Hey @rpatel15-hue. I wasn't able to reproduce the error, and I don't see any problems with the certificate. Sometimes a particular network can cause this type of problem. Can you ask them to try accessing the URL from a different network, if possible?
